jest/setupEnv.js

```diff
@@ -1,4 +1,5 @@
 require("babel-polyfill");
+require("reflect-metadata");
```

Comment:
This is a fragment of the plugins POC, right?

Reply:
It actually should be there already since we're using inversify.